app/controllers/buses_controller.rb

```diff
@@ -5,6 +5,11 @@ class BusesController < ApplicationController
   def index 
     @bus = Bus.all 
   end
+ 
+  def show
+    @route = Route.find(params[:id])
```

Comment:
why you use here instance variable route

Reply:
@route is an instance variable that stores the fetched Route object. Instance variables are available in the view corresponding to this controller action.